src/main/webapp/app/modules/home/ActivityElement.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Row, Col, Card, CardText, CardBody, CardTitle, CardGroup, Badge } from 'reactstrap';
+import { Translate, TextFormat } from 'react-jhipster';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { APP_DATE_FORMAT } from 'app/config/constants';
+
+const ActivityElement = props => (
+  <Row style={{ marginBottom: '15px' }}>
+    <Col>
+      <CardGroup>
+        <Card style={{ marginBottom: '0px' }}>
+          <CardBody>
+            <CardTitle style={{ color: '#66c4ff' }}>
```

Comment:
We should avoid inline styles, especially with color values. Please use scss with variables here

Reply:
ok